codegen/projections/high_score_service/lib/high_score_service/stubs.rb

```diff
@@ -43,7 +60,25 @@ def self.stub(http_resp, stub:)
       end
     end
 
-    # Operation Stubber for GetHighScore
+    class ErrorMessages
+      def self.default(visited=[])
```

Comment:
Unrelated to this change: Should we be using a `Set` for visited?  Probably yes....

Reply:
With the logic here, I don't think it's possible for the same key to exist twice, right? Because we check if it includes. What's the argument for Set?

Reply:
Just an argument for efficiency and using a data structure that matches the problem.

Reply:
Yeah, set can make sense. It would have to be required in this file.